website/static/js/myProjects.js

```diff
@@ -1324,6 +1324,30 @@ var Filters = {
  */
 var Information = {
     view : function (ctrl, args) {
+        function categoryMap(category) {
```

Comment:
so that the default logic is done here rather than something like:
` CATEGORY_MAP['analysis'] if CATEGORY_MAP['analysis']  else 'Uncategorized' `